sdk/servicebus/service-bus/test/README.md

```diff
@@ -2,7 +2,9 @@
 
 - Run `npm i` to install all the dependencies of this project. This is a one time task.
 - The tests expect a series of queues, topics and subscriptions to exist in a single Service Bus instance.
```

Comment:
After merging the `feature/service-bus-atom` branch, all tests now create the entities they need. So the above statement is no longer true.

```suggestion
- The tests will create the required queue, topic and subscription in the Service Bus namespace you provide using environment variables.
```